Automattic/wp-calypso
# Domains: Don't allow submitting transfer in support sessions

Disallow support to submit actual transfer requests, as this has compliance implications.

Test:
- SU as a user
- Go to the transfer screen
- Enter a domain
- Get to precheck screen
- Enter a valid auth code
- Confirm it
- Button always disabled
- See notice on top
